code-challenges.js

```diff
@@ -20,6 +31,9 @@ var temp3 = 212
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
+  for(let i= 0; i < myNumbers1.length; i++) {
+    console.log(myNumbers1[i] * 5)
+  }
```

Comment:
I'd double check the requirements on this challenge, it's asking for the for loop to be inside of a function. As it stands right now, the for-loop is just kind of floating out in the ether outside of a function.